cw721-timelock/src/state.rs

```diff
@@ -0,0 +1,30 @@
+use cosmwasm_std::{Addr};
+use cosmwasm_schema::cw_serde;
+use cw_storage_plus::Map;
+
+use andromeda_std::common::milliseconds::MillisecondsExpiration;
+
+// pub const CONFIG: Item<State> = Item::new("config");
+pub const TIMELOCKS: Map<&str, TimelockInfo> = Map::new("timelocks");
```

Comment:
Would a tuple key not be more appropriate here? Something like `(contract_address, token_id)`?

Reply:
Solved this problem

Reply:
This looks like it hasn't been resolved

Reply:
I solved this at next commit

Reply:
https://github.com/andromedaprotocol/ado-database/pull/9/commits/e86b8db844c7cd03bdb23438d5052fa960e6888e